setup.py

```diff
@@ -21,6 +21,9 @@
         'pyramid',
         'six',
     ],
+    tests_require=[
```

Comment:
Why not bump the version of py_zipkin in `install_requires`? I'm assuming we'd release a new version of pyramid_zipkin

Reply:
This is not technically a backwards incompatible change unless you use the `firehose_handler`, so I figured that there's no reason to add that requirement. Do you think we should?

Reply:
I think we should still bump the version just so that there isn't a discrepancy between what tests require and what the source requires. As a client developer, I would expect that if the feature is present in the current version of the library that I'm using, its minimum requirement specs are sufficient (even though you've mentioned it in the README)

Reply:
Tests automatically run with the latest versions of all dependencies. If this means the library is not backward compatible add it to install_requires, if not I'd simply delete this.

Reply:
TIL :) less of a reason to release a new version of pyramid_zipkin.

Still, because it depends on a newer version of a dependency, I'd say this is backwards compatible, but would require either a PATCH or MINOR version release

Reply:
pyramid_zipkin will definitely get a version bump but, we're <1.0.0 so semver is not really that strict